server/controllers/auth/signupHelpers/createParent.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  createUser, createParent, findUserByEmail, createStudent,
+} from '../../../queries';
+import { CustomError, UserTableInterface } from '../../../utils';
+
+const createParentAccount = async (user: UserTableInterface, children?: Array<string>) => {
+  const parentUser = await createUser(user);
+  const parentId = parentUser.getDataValue('id');
+
+  children?.forEach(async (child) => {
+    const doesChildStudent = await findUserByEmail(child);
+    const childId = doesChildStudent?.getDataValue('id');
+
+    if (!doesChildStudent) {
+      throw new CustomError(422, 'The email does not exist!');
+    }
+
+    await createStudent(childId, parentId);
```

Comment:
The student will be created already but we need to update the its parent

Reply:
The student will be created in the users table, but not in the students table, like you said to me.